21-web-game.js

```diff
@@ -0,0 +1,203 @@
+document.addEventListener("DOMContentLoaded", () => {
+    
+    let deck_id;
+    
+    let playerScore = 0;
+    let dealerScore = 0;
+    let buttonsContainer = document.querySelector("#playerButtons");
+    buttonsContainer.style.display = "none";
+    let playerHand = document.querySelector("#playerHand");
+    let dealerHand = document.querySelector("#dealerHand");
+    let urls = [];
+
+    let startBtn = document.querySelector("#startBtn");
+    
+    const fetchDeck = async () => {
+        try {
+            let res = await axios.get("https://deckofcardsapi.com/api/deck/new/");
+            deck_id = res.data.deck_id;
+            let shuffled = await axios.get(`https://deckofcardsapi.com/api/deck/${deck_id}/shuffle/`);
```

Comment:
Since you're not using the variable `shuffled` anywhere you can just `await axios.get(url)`